plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/MetadataEntry.java

```diff
@@ -173,6 +176,26 @@ public Optional<Boolean> isChangeDataFeedEnabled()
         return Optional.of(changeDataFeedEnabled);
     }
 
+    @JsonIgnore
+    public Map<String, Integer> getBaseFieldsPositionMap()
```

Comment:
Consider adding a unit test for this method once you're sure that it is actually needed.

Reply:
Removed. Not needed now.